src/code/code_8012EC80.c

```diff
@@ -725,3 +725,591 @@ void Inventory_SaveLotteryCodeGuess(PlayState* play) {
+        true,  // ITEM_CHATEAU
+        true,  // ITEM_HYLIAN_LOACH
+        true,  // ITEM_OBABA_DRINK
+        true,  // ITEM_MOONS_TEAR
+        true,  // ITEM_DEED_LAND
+        true,  // ITEM_DEED_SWAMP
+        true,  // ITEM_DEED_MOUNTAIN
+        true,  // ITEM_DEED_OCEAN
+        true,  // ITEM_ROOM_KEY
+        true,  // ITEM_LETTER_MAMA
+        true,  // ITEM_LETTER_TO_KAFEI
+        true,  // ITEM_PENDANT_OF_MEMORIES
+        true,  // ITEM_TINGLE_MAP
+        true,  // ITEM_MASK_DEKU
+        true,  // ITEM_MASK_GORON
+        true,  // ITEM_MASK_ZORA
+        true,  // ITEM_MASK_FIERCE_DEITY
+        false, // ITEM_MASK_TRUTH
+        false, // ITEM_MASK_KAFEIS_MASK
+        false, // ITEM_MASK_ALL_NIGHT
+        false, // ITEM_MASK_BUNNY
+        false, // ITEM_MASK_KEATON
+        false, // ITEM_MASK_GARO
+        false, // ITEM_MASK_ROMANI
+        false, // ITEM_MASK_CIRCUS_LEADER
+        false, // ITEM_MASK_POSTMAN
+        false, // ITEM_MASK_COUPLE
+        false, // ITEM_MASK_GREAT_FAIRY
+        false, // ITEM_MASK_GIBDO
+        false, // ITEM_MASK_DON_GERO
+        false, // ITEM_MASK_KAMARO
+        false, // ITEM_MASK_CAPTAIN
+        false, // ITEM_MASK_STONE
+        false, // ITEM_MASK_BREMEN
+        false, // ITEM_MASK_BLAST
+        false, // ITEM_MASK_SCENTS
+        false, // ITEM_MASK_GIANT
+        false, // ITEM_BOW_FIRE
+        false, // ITEM_BOW_ICE
+        false, // ITEM_BOW_LIGHT
+        false, // ITEM_SWORD_KOKIRI
+        false, // ITEM_SWORD_RAZOR
+        false, // ITEM_SWORD_GILDED
+        false, // ITEM_SWORD_DEITY
+        false, // ITEM_SHIELD_HERO
+        false, // ITEM_SHIELD_MIRROR
+        false, // ITEM_QUIVER_30
+        false, // ITEM_QUIVER_40
+        false, // ITEM_QUIVER_50
+        false, // ITEM_BOMB_BAG_20
+        false, // ITEM_BOMB_BAG_30
+        false, // ITEM_BOMB_BAG_40
+        false, // ITEM_WALLET_DEFAULT
+        false, // ITEM_WALLET_ADULT
+        false, // ITEM_WALLET_GIANT
+        false, // ITEM_FISHING_ROD
+        false, // ITEM_REMAINS_ODOLWA
+        false, // ITEM_REMAINS_GOHT
+        false, // ITEM_REMAINS_GYORG
+        false, // ITEM_REMAINS_TWINMOLD
+        false, // ITEM_SONG_SONATA
+        false, // ITEM_SONG_LULLABY
+        false, // ITEM_SONG_NOVA
+        false, // ITEM_SONG_ELEGY
+        false, // ITEM_SONG_OATH
+        false, // ITEM_SONG_SARIA
+        false, // ITEM_SONG_TIME
+        false, // ITEM_SONG_HEALING
+        false, // ITEM_SONG_EPONA
+        false, // ITEM_SONG_SOARING
+        false, // ITEM_SONG_STORMS
+        false, // ITEM_SONG_SUN
+        false, // ITEM_BOMBERS_NOTEBOOK
+        false, // ITEM_SKULL_TOKEN
+        false, // ITEM_HEART_CONTAINER
+        false, // ITEM_HEART_PIECE
+        false, // ITEM_71
+    },
+    // PLAYER_FORM_DEKU
+    {
+        true,  // ITEM_OCARINA_OF_TIME
+        false, // ITEM_BOW
+        false, // ITEM_ARROW_FIRE
+        false, // ITEM_ARROW_ICE
+        false, // ITEM_ARROW_LIGHT
```

Comment:
Question: what are these three items? If you enable these it doesn't allow the form to use the three named items, instead if you wanted to enable fire ice and light arrows you need to enable `ITEM_BOW_FIRE`/ICE/LIGHT instead.

Reply:
the arrows by themselves are their own items for the sake of sitting individually in the pause menu, and for collecting them as get items. they only become the "bow" variant when you equip them and they go on your c button.

Its possible to get the individual arrows on your c buttons too with glitches, and they behave normally.